Tweaks to template to use the string and endpoint

chrisreddington opened the pull request:
Tweaks to the template/tutorial to not use the object, and instead directly point to the blob endpoint and ensure consistency with the rest of the guide

azurecla commented:
Hi __@gbellmann__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
@gbellmann Thanks for the contribution! I'll have a member of our team review.

sethmanheim commented:
Hi, thanks for the feedback. This file is in the process of being renamed, but it's fine to make these changes now. Thanks again.

chrisreddington committed:
Tweaks to template to use the string and endpoint

Tweaks to the template/tutorial to not use the object, and instead directly point to the blob endpoint
